cli_tools_tests/common/config/config.go

```diff
@@ -0,0 +1,36 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
With the new folder structure, i.e.:
cli_tools_tests/module
cli_tools_tests/e2e

Where each subfolder holds certain category of tests, common seems out of place, as it could be mistaken for 'common' tests, not common code shared between different test categories. 

Is there overlap between common code in e2e and module tests? If not, we can split the code and move it into e2e/common and module/common. If there is an overlap, renaming common to indicate it doesn't actually hold tests is an option.

Reply:
Copying to https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1381